pkg/asset/installconfig/powervs/client.go

```diff
@@ -510,6 +555,24 @@ func (c *Client) GetPublicGatewayByVPC(ctx context.Context, vpcName string) (*vp
 	return nil, nil
 }
 
+// GetVPCSubnets retrieves all subnets in the given VPC.
+func (c *Client) GetVPCSubnets(ctx context.Context, vpcID string) ([]vpcv1.Subnet, error) {
+	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
```

Comment:
ditto

Reply:
Switched to ListSubnetsWithContext and passed along `ctx` instead of creating a child one.